lib/index.js

```diff
@@ -635,6 +658,10 @@ FSTree.prototype.symlinkSync = function(target, relativePath /*, type */) {
   aware, which is why it is an error to change the root of a non-source tree.
 */
 FSTree.prototype.reread = function(newRoot) {
+  if(newRoot && path.resolve(newRoot) !== newRoot) {
+    newRoot =  path.resolve(newRoot);
+  }
```

Comment:
i don't think we need to `path.resolve` twice.  it seems clearer to just

```js
if(newRoot) { newRoot = path.resolve(newRoot) + path.sep; }
```

we should then be able to drop the subsequent calls to `normalize` as well as `resolve` will also `normalize`